docs/source/index.rst

```diff
@@ -25,6 +25,34 @@ You can install directly from the source on github by following these steps:
 
       git checkout $branch
 
+
+Ansible on RHEL and CentOS
+--------------------------
+You can acquire Ansible on RHEL and CentOS by installing from
+`"Extras"<https://access.redhat.com/solutions/912213>`_.
+
+On RHEL::
+
+    subscription-manager repos --enable=rhel-7-server-extras-rpms
+
+(CentOS does not use subscription-manager and already has "Extras" enabled by default.)
```

Comment:
I saw your other PR that talks abut ansible version 2.3.1, I believe the latest centos 7.3 and 7.4 would have that as default.